Frontend/ui-common/src/Engineering/Helpers/MobilityPrelude.purs

```diff
@@ -0,0 +1,52 @@
+module Engineering.Helpers.MobilityPrelude where
+
+import Data.String (null,Pattern(..), contains, joinWith, toLower, take, toUpper, drop)
+import Data.Maybe (Maybe(..), fromMaybe)
+import PrestoDOM as PD
+import Prelude
+import Data.Array (elem)
+
+isStrEmpty :: String -> Boolean
+isStrEmpty = null
+
+has :: String -> String -> Boolean
+has msg errorPattern = contains (Pattern errorPattern) msg
+
+equalIgnoreCase :: String -> String -> Boolean
+equalIgnoreCase val1 val2 = (toLower val1) == (toLower val2)
+
+isPresent :: String -> Array String -> Boolean
+isPresent key array =
+      (toLower key) `elem` (map (\item -> toLower item) array)
```

Comment:
what if exact match is required? might not help in our usecase. We can remove

Reply:
Should I rename fiction ? Added few extra functions which we might need later